Sources/Screens/Home/TabbedViewController.swift

```diff
@@ -0,0 +1,104 @@
+import Coordination
+import GDSCommon
+import UIKit
+
+final class TabbedViewController: BaseViewController {
+
+    override var nibName: String? { "TabbedView" }
+    @IBOutlet private var tableView: UITableView! {
+        didSet {
+            tableView.accessibilityIdentifier = "tabbed-view-table-view"
+        }
+    }
+    
+    private let headerView: UIView?
+    private let viewModel: TabbedViewModel
+    private var accessToken: String?
+    
+    init(viewModel: TabbedViewModel,
+         headerView: UIView? = nil) {
+        self.viewModel = viewModel
+        self.headerView = headerView
+        super.init(viewModel: viewModel,
+                   nibName: "TabbedView",
+                   bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        tableView.register(TabbedTableViewCell.self, forCellReuseIdentifier: "tabbedTableViewCell")
+        tableView.tableHeaderView = headerView
+        tableView.delegate = self
+        tableView.dataSource = self
+        title = viewModel.navigationTitle?.value
+        
+    }
+    
+    override func viewIsAppearing(_ animated: Bool) {
+        navigationController?.navigationBar.prefersLargeTitles = true
+        navigationController?.navigationBar.sizeToFit()
+    }
+
+    override func viewDidLayoutSubviews() {
+        super.viewDidLayoutSubviews()
+        resizeHeaderView()
+    }
+    
+    func updateToken(accessToken: String?) {
+        // To be replaced by secure token JWT with capability to extract e-mail for display
+        self.accessToken = accessToken
+        guard let headerView = headerView as? SignInView else { return }
+        headerView.updateEmail("sarahelizabeth_1991@gmail.com")
+        resizeHeaderView()
+    }
```

Comment:
I imagine we can do away with (or rename/repurpose) this method now that we don't care about exposing the token on the home page? (Perhaps repurposing would be better if we think that updating the header is needed here)